README.md

```diff
@@ -415,6 +431,7 @@ API | Description | Auth | HTTPS | Link |
 | Songsterr | Provides guitar, bass and drums tabs and chords | No | Yes | [Go!](https://www.songsterr.com/a/wa/api/) |
 | Soundcloud | Music | No | Yes | [Go!](https://developers.soundcloud.com/) |
 | Spotify | Music | `OAuth` | Yes | [Go!](https://developer.spotify.com/web-api/) |
+| TasteDive | Similar artist API (also works for movies and TV shows) | `apiKey` | Yes | [Go!](https://tastedive.com/read/api) |
```

Comment:
Doesn't specify if the TasteDive service is free. Can you confirm?

Reply:
It is free. I was able to sign up and get a working key with no fee paid.

Reply:
Is there documentation somewhere?

Reply:
@davemachado no, I do not see the price documented anywhere.

Reply:
OK I googled around and things look good!